### ISSUE TITLE
Is there a way we can make those links accessible from outside of our campus ?

### ISSUE THREAD

**Issue by AnshulBasia** [coderkd10/IITD_User_Pages#1]

**Comment by coderkd10** [comment]
Yes, this project is intended to make links from `privateweb.iitd.ac.in` public. But for that you need to copy `user.php` (and get its dependencies via composer or otherwise) to a public facing internal server.

This project was intended for use by the Electrical Engineering Society (EES) who wanted to provide public web-pages to EE students akin to what CS department does. They deployed it for sometime but took it down to confirm from CSC if doing this was ok. Haven't heard back from them since then. Maybe you can ping the G.Sec?
